src/App.js

```diff
@@ -0,0 +1,21 @@
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+import Rockets from './components/Rockets';
+import Missions from './components/Missions';
+import MyProfile from './components/MyProfile';
+import Navbar from './components/Navbar';
+
+function App() {
+  return (
+    <main className="App">
+      <Navbar />
+      <Routes>
+        <Route element={<Rockets />} path="/rockets" />
+        <Route element={<Missions />} path="/missions" />
+        <Route element={<MyProfile />} path="/" />
```

Comment:
- Hey Team, According to the project requirement below :point_down: 

```
Rockets is the default view, so its data would be fetched when the application starts. However, the missions data should only be fetched (once) when a user navigates to the Missions section. (the same thing for Dragons)
```

Please, kindly ensure that the rocket is the default view as you've made my profile the default view which does not meet this part of the project requirement. Please, kindly fix this.